src/components/segmentedControl/index.tsx

```diff
@@ -9,17 +9,26 @@ import Reanimated, {
   withTiming,
   runOnJS
 } from 'react-native-reanimated';
-import {Colors, BorderRadiuses, Spacings} from '../../style';
+import {Colors} from '../../style';
 import {Constants, asBaseComponent} from '../../commons/new';
 import View from '../view';
 import Segment, {SegmentedControlItemProps} from './segment';
+import useSegmentedControlPreset from './useSegmentedControlPreset';
+import {ComponentStatics} from '../../typings/common';
 
-const BORDER_WIDTH = 1;
+const CONTAINER_BORDER_WIDTH = 1;
 const TIMING_CONFIG = {
   duration: 300,
   easing: Easing.bezier(0.33, 1, 0.68, 1)
 };
 
+const presets = {
```

Comment:
What's wrong with a simple enum?

Reply:
In this case It doesn't really have any effects. If we would do something like this we get ts errors:
![image](https://github.com/wix/react-native-ui-lib/assets/145321715/275b8694-a264-4598-94cb-c9cafef1877a)